source/user-manual/reference/ossec-conf/syscheck.rst

```diff
@@ -815,6 +877,8 @@ Specifies the maximum number of seconds between every inventory synchronization.
 
 **response_timeout**
 
+.. deprecated:: 4.4
```

Comment:
Let's remove all the setting descriptions for the deprecated ones. That way the reference keeps cleaner. In addition, they can be read by selecting a previous version.